org.openkj.OpenKJ.yml

```diff
@@ -1,19 +1,21 @@
 app-id: org.openkj.OpenKJ
 runtime: org.kde.Platform
-runtime-version: 5.15
+runtime-version: 5.15-24.08
 sdk: org.kde.Sdk
 command: openkj
-finish-args: 
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+  org.freedesktop.Platform.GStreamer.gstreamer-vaapi:
```

Comment:
this was wrong, it will not work.

Reply:
See https://docs.flatpak.org/en/latest/extension.html#loading-existing-extensions